shared/middleware/helpers/firebase_helper.rb

```diff
@@ -51,8 +51,16 @@ def csv_as_table(csv_data)
     return records, columns
   end
 
+  def escape_table_name(table_name)
+    return URI.escape(table_name).gsub('.', '%252E')
```

Comment:
Is `%252E` a typo? (i.e. other places it's `%2E`)

Reply:
Nope, here we need to escape the `%` as well, so `%2E` would end up as just `2E` and `%252E` ends up being `%2E`.

Reply:
👍